### PR TITLE
#21349: Refactor long functions in entrynodes.c (2)

### PR DESCRIPTION
**Pull request by rl1987**
https://trac.torproject.org/projects/tor/ticket/21349

### CODE REVIEW

File: src/or/entrynodes.c [hunk 1]
```diff
@@ -407,6 +404,13 @@ get_remove_unlisted_guards_after_days(void)
                                  DFLT_REMOVE_UNLISTED_GUARDS_AFTER_DAYS,
                                  1, 365*10);
 }
+
+static time_t
```
Comment (asn-d6) [inline src/or/entrynodes.c]: Please add the trivial func doc here.

File: src/or/entrynodes.c [hunk 2]
```diff
@@ -1411,11 +1413,11 @@ sampled_guards_update_from_consensus(guard_selection_t *gs)
     approx_time() - get_guard_confirmed_min_lifetime();
 
   /* Then: remove the ones that have been junk for too long */
+  n_changes +=
   sampled_guards_prune_obsolete_entries(gs,
```
Comment (asn-d6) [inline src/or/entrynodes.c]: Please intend the code here! :)

File: src/or/entrynodes.c (hunk 1 above)
Comment (rl1987) [reply src/or/entrynodes.c]: Fixed - 9a3d655a2a1a8068b25b35494f00ceb805977b6d.

File: src/or/entrynodes.c (hunk 2 above)
Comment (rl1987) [reply src/or/entrynodes.c]: Fixed in 870dab729af6bec6f1d01378a104ba8f25490b74.

**Comment by tlyu** [comment]
Superseded by #246.

**Pull request closed without merging**